src/test/java/seedu/address/logic/commands/AddSpecialisationCommandTest.java

```diff
@@ -0,0 +1,182 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SPECIALISATION_1;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_SPECIALISATION_2;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showDoctorAtIndex;
+import static seedu.address.testutil.TypicalDatabase.getTypicalDatabase;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+
+import java.util.Objects;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Database;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.doctor.Doctor;
+import seedu.address.model.tag.Tag;
+import seedu.address.testutil.DoctorBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for AddDoctorSpecialisationCommand.
```

Comment:
`AddDoctorSpecialisationCommand ` change to `AddSpecialisationCommand `

Reply:
Done, thanks